src/component/Header.css

```diff
@@ -16,5 +16,16 @@
     text-decoration: none;
     color: #ffffff;
     font-weight: bold;
-    font-size: 1.5em;
+    font-size: 1.1em;
+}
+.container {
```

Comment:
This affects all containers, should we add a more specific css rule instead? We may not even need to use `!important` at all if we have a more specific rule.